src/screens/Followings/FollowersScreen.js

```diff
@@ -103,7 +114,8 @@ function FollowersScreen() {
 
       <Followings
         isLoading={isLoading}
-        dataFollower={dataFollower}
+        dataFollower={followedUsers}
+        dataUnfollowed={unfollowedUsers}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `Followings` component now receives two new props: `dataFollower` and `dataUnfollowed`. This change aligns with the previous filtering of follower data into `followedUsers` and `unfollowedUsers`.
```javascript
<Followings
  isLoading={isLoading}
  dataFollower={followedUsers}
  dataUnfollowed={unfollowedUsers}
  setDataFollower={setDataFollower}
/>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->